docker_tests/test_prod_image.py

```diff
@@ -158,6 +158,7 @@ def test_pip_dependencies_conflict(self):
         "grpc": ["grpc", "google.auth", "google_auth_httplib2"],
         "hashicorp": ["hvac"],
         "ldap": ["ldap"],
+        "mysql": ["MySQLdb", *(["mysql"] if bool(find_spec("mysql")) else [])],
```

Comment:
We should always check `import MySQLdb` which should prevent regression

Reply:
Hmm.. Should we rebuild 2.8.0 images I guesss (I see that the released images are failing - but we can rebuild and repush the images with Mariadb, that should be easy.

Reply:
Yeah I think it would be the best option, rather than think provide workaround how to properly remove MySQL libraries and replace them by MariaDB. 

Otherwise we would have multiple issues everywhere: Github / Slack / SO. 
Seems like all holidays is over and users started to upgrade their Airflow instances 🤣

Reply:
Right - we can easily rebuild the images using `main` Dockerfile, so let me merge it and rebuild the images.